node_interop/lib/atomics.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020, Anatoly Pulyaevskiy. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+/// Bindings for the Atomics JavaScript obect. Since this object only has static
+/// methods, it's exposed as a module so that it can be used in the standard
+/// Dart style.
```

Comment:
One thing I prefer to preserve, if possible, is to keep bindings closer to the actual JS API. What do you think of adding a container `Atomics` abstract class and making the below methods static inside it? I _think_ JS interop supports external static methods, but I'm not 100% sure.